swedish-oidc-attribute-specification.md

```diff
@@ -357,55 +363,31 @@ For each scope defined below a set of claims is declared. Each declared claim ha
-| `given_name` | Given name | Mandatory |
-| `https://claims.oidc.se/1.0/country` | Country code for the eIDAS country that authenticated the subject. | Mandatory |
-| `txn` | ID of assertion issued by the member state node. | Mandatory |
-| `https://claims.oidc.se/`<br />`1.0/personalNumber` | The Swedish civic registration number of the subject. This claim is delivered if the Swedish eIDAS node can find a registered binding between the eIDAS person identifier and a Swedish civic registration number. | Optional |
-| `https://claims.oidc.se/1.0/`<br />`eidas/personalNumberBinding` | If the Swedish civic registration number is released, this claim is also included to indicate under which process the mapping between the eIDAS person identifier and the Swedish civic registration number was performed. | Optional |
-
-The Swedish eIDAS node can also deliver additional claims that need to be explicitly requested by the relying party. These claims are then delivered if available from the foreign member state node. The claims are:
-
-- `https://claims.oidc.se/1.0/birthName` - Birth name of the subject.
-- `https://claims.oidc.se/1.0/placeOfbirth` - Place of birth of the subject.
-- `gender` - Gender of the subject.
-- `address` - Current address of the subject.
+> **\[\*\]**: A `personalNumber` OR `coordinationNumber` claim MUST be delivered, but not both.
 
 <a name="natural-person-organizational-identity"></a>
-### 3.4. Natural Person Organizational Identity
+### 3.3. Natural Person Organizational Identity
 
 **Scope:** `https://scopes.oidc.se/1.0/naturalPersonOrgId`
 
 **Description:** The “Natural Person Organizational Identity” scope provides basic organizational identity information about a person. The organizational identity does not necessarily imply that the subject has any particular relationship with or standing within the organization, but rather that this identity has been issued/provided by that organization for any particular reason (employee, customer, consultant, etc.).
 
 | Claim | Description/comment | Requirement |
 | :--- | :--- | :--- |
-| `name` | Display name. The claim MAY contain personal information such as the given name or surname, but it MAY also be used as an anonymized display name, for example, "Administrator 123". This is decided by the issuing organization. | Mandatory |
-| `https://claims.oidc.se/`<br />`1.0/orgAffiliation` | Personal identifier and organizational number (`https://claims.oidc.se/1.0/orgNumber`). | Mandatory |
-| `https://claims.oidc.se/`<br />`1.0/orgName` | Organization name | Mandatory |
+| `name` | Display name. The claim MAY contain personal information such as the given name or surname, but it MAY also be used as an anonymized display name, for example, "Administrator 123". This is decided by the issuing organization (\[[OpenID.Core](#openid-core)\]). | Mandatory |
+| `https://claims.oidc.se/`<br />`1.0/orgAffiliation` | Personal identifier and organizational number. | Mandatory |
```

Comment:
Possibly elaborate slightly on the orgAffiliation attribute? I was not familiar with it and was somewhat confused given its description as `Personal identifier AND organizational number` (my emphasis). 

I looked it up at [Sweden Connect](https://docs.swedenconnect.se/technical-framework/latest/04_-_Attribute_Specification_for_the_Swedish_eID_Framework.html#the-orgaffiliation-attribute) which clarified its intent and usage. Perhaps include a reference to the oidc.se claim-mapping for this attribute?

Reply:
Good point. I will elaborate on the claim here, but not make any references to Sweden Connect. Don't want circular refs in profiles.